### PR TITLE
Expressify api-example apps

### PR DESCRIPTION
**Pull request by gadenbuie**
Tracks ~all~ a big chunk of the work for #987

### CODE REVIEW

File: shiny/api-examples/layout_column_wrap/app-express.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import shiny
+from shiny.express import ui
+
+a_card = shiny.ui.card("A simple card")
```
Comment (cpsievert) [inline shiny/api-examples/layout_column_wrap/app-express.py:4]:
```suggestion
from shiny.express import ui

with ui.hold() as a_card:
    with ui.card():
        "A simple card"
```

File: shiny/api-examples/markdown/app-express.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from shiny.express import input, ui
```
Comment (cpsievert) [inline shiny/api-examples/markdown/app-express.py:1]:
```suggestion
from shiny.express import ui
```

File: shiny/api-examples/layout_column_wrap/app-express.py (hunk 1 above)
Comment (gadenbuie) [reply shiny/api-examples/layout_column_wrap/app-express.py:4]: oh thanks, sorry I just pushed work I had started on a week ago

Comment (gadenbuie) [reply shiny/api-examples/layout_column_wrap/app-express.py:4]: a lot has changed in a week!

File: shiny/api-examples/layout_sidebar/app-express.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from shiny import render
+from shiny.express import input, ui
```
Comment (gadenbuie) [inline shiny/api-examples/layout_sidebar/app-express.py:5]:
```suggestion
from shiny.express import input, render, ui
```

File: shiny/api-examples/page_sidebar/app-express.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from shiny import render
+from shiny.express import input, ui
```
Comment (gadenbuie) [inline shiny/api-examples/page_sidebar/app-express.py:5]:
```suggestion
from shiny.express import input, render, ui
```

File: shiny/api-examples/sidebar/app-express.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
+from shiny import render
+from shiny.express import input, ui
```
Comment (gadenbuie) [inline shiny/api-examples/sidebar/app-express.py:2]:
```suggestion
from shiny.express import input, render, ui
```

File: shiny/api-examples/panel_conditional/app-express.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+from shiny.express import input, ui
```
Comment (cpsievert) [inline shiny/api-examples/panel_conditional/app-express.py:1]:
```suggestion
from shiny.express import ui
```

**Comment by gadenbuie** [comment]
Manually merged into #1073

**Pull request closed without merging**